dhis-2/dhis-services/dhis-service-reporting/src/main/java/org/hisp/dhis/validation/PeriodTypeExtended.java

```diff
@@ -81,6 +82,12 @@
 
     private boolean nonSlidingWindowsNeeded = false;
 
+    private Set<DimensionalItemId> leftSideItemIds = new HashSet<>();
+
+    private Set<DimensionalItemId> rightSideItemIds = new HashSet<>();
+
+    private Map<DimensionalItemId, DimensionalItemObject> dimensionItemMap;
```

Comment:
Feels like a mix of styles where the added sets are initialised and the manipulated in place while the map is set via setter.

Reply:
Yup. Actually, you caught me here. I had added this field during the process of refactoring the code for this ticket, but eventually I added it to ValidationRunContext instead, because that better fit the logic. But I forgot to remove it from here, and it was unused. I've removed it now, which also removes the stylistic inconsistency. Thanks again.